Jenkinsfile_CNP

```diff
@@ -7,13 +7,13 @@ def product = "am"
 def component = "lib-test"
```

Comment:
why was this file touched? I would revert

Reply:
👍